src/core_plugins/kibana/public/management/sections/objects/change_index_modal.js

```diff
@@ -0,0 +1,211 @@
+    const result = Object.keys(this.state.objects).map(indexPatternId => ({
+      oldId: indexPatternId,
+      newId: this.state.objects[indexPatternId].newIndexPatternId,
+    }));
+    this.props.onChange(result);
+  };
+
+  onIndexChange = (id, event) => {
+    event.persist();
+    this.setState(state => {
+      return {
+        objects: {
+          ...state.objects,
+          [id]: {
+            ...state.objects[id],
+            newIndexPatternId: event.target.value,
+          }
+        }
+      };
+    });
+  };
+
+  onKeyDown = (event) => {
+    if (event.keyCode === keyCodes.ESCAPE) {
+      this.props.onClose();
+    }
+  };
+
+  render() {
+    const { page, perPage } = this.state;
+    const totalIndexPatterns = Object.keys(this.state.objects).length;
+    const indexPatternIds = Object.keys(this.state.objects).slice(page, page + perPage);
+    const rows = indexPatternIds.map((indexPatternId, key) => {
+      const objects = this.state.objects[indexPatternId].list;
+      const sample = take(objects, 5).map((obj, key) => <span key={key}>{obj.name}<br/></span>);
+
+      return (
+        <KuiTableRow key={key}>
+          <KuiTableRowCell>
+            {indexPatternId}
+          </KuiTableRowCell>
+          <KuiTableRowCell>
+            {objects.length}
+          </KuiTableRowCell>
+          <KuiTableRowCell>
+            {sample}
+          </KuiTableRowCell>
+          <KuiTableRowCell>
+            <select
+              className="kuiSelect kuiSelect--medium"
+              data-test-subj="managementChangeIndexSelection"
+              value={this.state.objects[indexPatternId].newIndexPatternId}
+              onChange={this.onIndexChange.bind(this, indexPatternId)}
+            >
+              <option value={indexPatternId}>-- None --</option>
+              {this.props.indices.map((index, i) => {
+                return (
+                  <option key={i} value={index.id}>
+                    {index.get('title')}
+                  </option>
+                );
+              })}
+            </select>
+          </KuiTableRowCell>
+        </KuiTableRow>
+      );
+    });
+
+    return (
+      <KuiModalOverlay>
+        <KuiModal
+          data-tests-subj="managementChangeIndexModal"
+          aria-label="Index does not exist"
+          className="managementChangeIndexModal"
+          onKeyDown={this.onKeyDown}
+        >
+          <KuiModalHeader>
+            <KuiModalHeaderTitle>
+              Index Pattern Conflicts
+            </KuiModalHeaderTitle>
+          </KuiModalHeader>
+          <KuiModalBody>
+            <KuiModalBodyText>
+              <p>
+                The following saved objects use index patterns that do not exist.
+                Please select the index patterns you&apos;d like re-associated them with.
+              </p>
+            </KuiModalBodyText>
+            <KuiControlledTable>
+              <KuiToolBar>
+                <KuiToolBarSection>
```

Comment:
Can we eliminate this row unless there are multiple pages?

Reply:
I tried it a few different ways and not sure what's the best visuals:

This is now:
<img width="801" alt="screen shot 2017-09-26 at 1 18 00 pm" src="https://user-images.githubusercontent.com/56682/30874087-5660f67c-a2bd-11e7-9daf-2ff26ee9c004.png">

This is if I don't show the toolbar:
<img width="802" alt="screen shot 2017-09-26 at 1 19 23 pm" src="https://user-images.githubusercontent.com/56682/30874097-5deb11b6-a2bd-11e7-94e6-bfcdb0ac1668.png">

The best way is probably to only do the controlled table but that seems like a weird thing to read later?

Reply:
It should definitely be possible to render a table without the toolbar at the top. You'll just have to take it out of the controlled table component. There should be an example in the doc site, I think?